examples/advice_provider.inputs

```diff
@@ -0,0 +1,19 @@
+{
+    "operand_stack": [],
+    "advice_stack": ["1", "2", "3", "4"],
+    "advice_map": {
+        "0000000000000000000000000000000000000000000000000000000000000000": [5, 6, 7, 8],
+        "0000001000000000000000000000000000000000000000000000000000000000": [9, 10, 11, 12]
```

Comment:
Do you think a test with hex format (i.e. `0xbeef`) will work here? If yes, we could add one entry with such format to demonstrate it is also possible.

Reply:
This wouldn't work.  The keys defined int he `.inputs` file must be 32 bytes long (64 hex characters).  We could maybe add some logic in the future to pad if this is not the case.